sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/ListStateImplWithTTL.scala

```diff
@@ -238,11 +217,21 @@ class ListStateImplWithTTL[S](
     }
   }
 
+  private[sql] def getMinValues(): Iterator[Long] = {
+    val groupingKey = stateTypesEncoder.encodeGroupingKey()
+    minIndexIterator()
+      .filter(_._1 == groupingKey)
+      .map { case _ @ (_, minExpirationMs) => minExpirationMs }
```

Comment:
nit: do we need to do this instead of just doing `_._2` while we do `_._1` just in the above (meaning we clearly know the tuple here)?